### ISSUE TITLE
Support Touch Events

### ISSUE THREAD

**Issue by marcojakob** [marcojakob/dart-html5-dnd#3]
Supporting touch events for HTML5 drag and drop would be great, although I'm not sure if this can and should be implemented for this library:
### Problems
1. On a touch device the drag gesture usually means scrolling the page. And there might also be difficulties with other gestures like pinch and zoom, etc.
2. Translate touch events into dragStart, drag, dragEnd, dragEnter, dragLeave, dragOver, drop events. (see [javascript drag and drop for touch devices](http://stackoverflow.com/a/6362527/862411) and [jquery-ui-touch-punch](https://github.com/yeco/jquery-ui-touch-punch))
3. It is possible to manually dispatch dragStart events but I couldn't find a way to make the browser generate the drag image. How do you "fake" drag events so that the browser actually recognizes them as drag events?

If nr. 3 is not possible, the only possibility would be to _fake everything_. That means we would need to manually translate touch events into all the drag and drop events (see nr. 2). Then, we'd need to fake the drag image by manually creating and moving an image (like we already do for the ie polyfill). And that's not all, there is also the drop effect which has impact on the mouse cursor, etc. 

In other words: The idea of this library is to use as much of the browser's native HTML5 drag and drop as possible. Replacing all HTML5 drag and drop elements with our own implementation wouldn't make sense. So, hopfully there is an answer to problem nr. 3.

**Comment by marcojakob** [comment]
Despite my initial concerns, I took a stab at supporting touch events. With v0.3.0 I replaced the workaround for IE9 with a complete emulation of HTML5 drag and drop. This means that we can use the same emulation for touch events pretty easily. Instead of listening to mouseDown, mouseMove and mouseUp events we would just be listening to touchStart, touchMove, touchEnd.

(And, I was wrong about impact on mouse cursor... There is no cursor on touch devices, of course.)

**Comment by marcojakob** [comment]
Touch support is ready for v0.3.1
